src/qibolab/instruments/abstract.py

```diff
@@ -97,6 +97,18 @@ def play(self, *args, **kwargs):
             the acquired :class:`qibolab.result.ExecutionResults` object.
         """
 
+    def split_batches(self, sequences):
+        """Split sequences to batches each of which can be unrolled and played as a single sequence.
+
+        Args:
+            sequence (list): List of :class:`qibolab.pulses.PulseSequence` to be played.
+
+        Returns:
+            Iterator of batches that can be unrolled in a single one.
+            Default will return all sequences as a single batch.
+        """
+        return [sequences]
```

Comment:
It is used by the dummy instrument. We could still turn to `@abstractmethod` but this would force implementing it to all instruments (which may not be needed if they don't care about unrolling for some reason).